src/accelerate/state.py

```diff
@@ -196,3 +198,14 @@ def __repr__(self):
     @property
     def use_fp16(self):
         return self.mixed_precision != "no"
+
+    def _check_initialized(self, mixed_precision=None, cpu=None):
+        "Checks if a modification is trying to be made and the `AcceleratorState` has already been initialized"
+        if getattr(self, "initialized", False):
+            template = Template(
+                "AcceleratorState has already been initialized and cannot be changed, restart your runtime completely and pass `$flag` to `Accelerate()`."
+            )
```

Comment:
I don't understand why we are using this class, when you can just do a regular string with flag between curly brackets and then use the format method.

Reply:
Definitely not because I forgot strings could do this... 👀